experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -592,6 +592,19 @@ export const getOutgoingRequestMetricAttributesOnResponse = (
   return metricAttributes;
 };
 
+/**
+ * Returns outgoing stable request Metric attributes scoped to the response data
+ * @param {Attributes} spanAttributes the span attributes
+ */
+export const getStableOutgoingRequestMetricAttributesOnResponse = (
+  spanAttributes: Attributes
+): Attributes => {
+  return {
```

Comment:
HTTP flavor is no longer being added in this case, looking at the docs this field was replaced by network.protocol.name, same for Network Peer port, this one is stable and with no change and not being returned anymore.

Reply:
We should some validation in the unit tests as well for those in the "stable" scenario

Reply:
Looks like the attributes are not part of HTTP Metrics semantic convention anymore, so resolving this.
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md